spec/02-integration/03-db/08-declarative_spec.lua

```diff
@@ -10,6 +10,12 @@ for _, strategy in helpers.each_strategy() do
     lazy_setup(function()
       local _
       _, db = helpers.get_db_utils(strategy)
+
+      -- This is a special case, where some DB states could be corrupted by DB truncation in `lazy_teardown()`.
+      -- We manually bootstrap the DB here to ensure the creation of a table is done correctly
+      db:schema_reset()
```

Comment:
Perhaps we can place `db:schema_reset()` before `get_db_utils` and also remove ` helpers.bootstrap_database(db)`.
bootstrap_database will be called in `get_db_utils`. https://github.com/Kong/kong-ee/blob/master/spec/helpers.lua#L584

Reply:
No, we must call `get_db_utils` first then get variable `db` to call other apis.